webapp/graphite/events/views.py

```diff
@@ -34,9 +34,17 @@ def view_events(request):
 
 
 def detail(request, event_id):
-    e = get_object_or_404(Event, pk=event_id)
-    context = {'event': e}
-    return render_to_response("event.html", context)
+    if request.META['HTTP_ACCEPT'] == 'application/json':
+        e = Event.objects.get(id=event_id)
+        e.tags = e.tags.split()
+        response = HttpResponse(
+            json.dumps(model_to_dict(e), cls=EventEncoder),
+            content_type="application/json")
+        return response
```

Comment:
`Django>=1.7` has a built-in JSON response, perhaps it'd be useful to have it in graphite-web to avoid calling json.dumps and passing content-type every time JSON is returned.